components/layout/fragment.rs

```diff
@@ -2602,11 +2602,6 @@ impl Overflow {
         self.paint = self.paint.union(&other.paint);
     }
 
-    pub fn union_rect(&mut self, rect: &Rect<Au>) {
```

Comment:
What shall we do here? For what it's worth, I agree with @nox. I think dead code should be avoided. If the function will be used in the future, it should be added at that moment, otherwise it's only confusing to have it around. I'm also fine with updating the patch to add the dead code annotation if you think that's better though, so please let me know how we shall proceed :-)